scripts/holes.pl

```diff
@@ -31,16 +31,15 @@ sub holes_sub {
     my $output = `$holes_cmd`;
     my $sep = Irssi::parse_special(Irssi::settings_get_str('holes_separator'));
     my @lines = split /\n/, $output;
-    my @lines = uniq(@lines);
+    @lines = uniq(@lines);
     $holes = '';
     $result = @lines;
     foreach my $line (@lines) {
-        if ($result == "") {
+        if ($result eq "") {
             $result = $line
         }
         $result = $result.$sep.$line
```

Comment:
not sure what you mean.

Reply:
aah, that only adds `$line` for the first element and not the `$separator`. so the line looks like this:
```
29 > 12344 > 14514 > 
```
instead of this:
```
> 29 > 12344 > 14514 >
```